text/exercise2/world_cup.md

```diff
@@ -41,3 +34,26 @@ In recent years, the Women's World Cup has witnessed growing support, with
 increased viewership, sponsorship, and media coverage, reflecting the 
 escalating interest and recognition of women's soccer.
 
+### Soccer vs. Football
+
+The difference between football and soccer primarily lies in regional
+terminology and the style of the game played.
+
+In most parts of the world, including Europe, South America, Africa, and 
+Asia, "football" refers to what Americans call "soccer." This version of 
+football involves two teams of 11 players each competing to score goals by 
+kicking a round ball into the opponent's net, primarily using feet, head, 
+or body (except for the goalkeeper) without handling the ball with hands 
+or arms during play, except for the goalkeeper within their penalty area.
+
+In the United States and Canada, "football" refers to a different sport 
+known as American or Canadian football. This game involves two teams of 11
+ players each trying to advance an oval-shaped ball down the field to score 
+touchdowns by carrying or passing it, with players wearing protective gear 
+and employing tactics involving both running and passing.
+
+The key distinctions include the shape of the ball, the style of play, the 
+rules, and the terminology used in different regions. Both football (soccer) 
+and American/Canadian football are immensely popular in their respective areas, 
+contributing to the global landscape of sports in their unique ways.
+
```

Comment:
A condensed version of this text could be moved to the beginning of the article, but this much background is unneccessary.